modules/puppet/files/usr/lib/nagios/plugins/check_puppet_agent

```diff
@@ -13,6 +15,13 @@ if File.file?(puppet_agent_disabled_lockfile)
   exit 1
 end
 
+hours_since_last_run = (Time.now - File.mtime(puppet_agent_summary_filename)) / 3600
```

Comment:
There is a small oddity here in that this file won't exist when the machine is built but before puppet runs. But as we deploy this test with puppet this'll be fine.